curriculum/challenges/english/20-upcoming-python/learn-algorithm-design-by-building-the-shortest-path-algorithm/65578fcf00322dbad5dee05b.md

```diff
@@ -0,0 +1,63 @@
+---
+id: 65578fcf00322dbad5dee05b
+title: Step 28
+challengeType: 20
+dashedName: step-28
+---
+
+# --description--
+
+When the distance of the neighbour node (the second item in the processed tuple) plus the distance of `current` is less than the currently known distance of the neighbour node (the first item in the processed tuple), you need to reassign the neighbour node distance to that sum. Create an `if` statement for that.
```

Comment:
I think, it would be better if this step is split into two steps:
- First step for building the condition
- Second for the body of the condition.